src/web/releases.rs

```diff
@@ -1213,4 +1213,57 @@ mod tests {
             Ok(())
         });
     }
+
+    #[test]
+    fn check_releases_page_content() {
+        let sel = ".pure-menu-horizontal>.pure-menu-list>.pure-menu-item>.pure-menu-link>.title";
+        wrapper(|env| {
+            let tester = |url| {
+                let page = kuchiki::parse_html()
+                    .one(env.frontend().get(url).send().unwrap().text().unwrap());
+                assert_eq!(page.select("#crate-title").unwrap().count(), 1);
+                let not_maching = page
+                    .select(sel)
+                    .unwrap()
+                    .map(|node| node.text_contents())
+                    .zip(
+                        [
+                            "Recent",
+                            "Stars",
+                            "Recent Failures",
+                            "Failures By Stars",
+                            "Activity",
+                            "Queue",
+                        ]
+                        .iter(),
+                    )
+                    .filter(|(a, b)| a.as_str() != **b)
+                    .collect::<Vec<_>>();
+                if !not_maching.is_empty() {
```

Comment:
nit: double negatives are hard to read
```suggestion
                if not_maching.len() > 0 {
```